### PR TITLE
Add more information to documentation about debugging

### PR DESCRIPTION
**Pull request by DanielBallaSZTE**
Some more information is added to the debugger documentation of IoT.js specific things, for clarity.

IoT.js-DCO-1.0-Signed-off-by: Daniel Balla dballa@inf.u-szeged.hu

### CODE REVIEW

File: docs/devs/Use-JerryScript-Debugger.md [hunk 1]
```diff
@@ -12,6 +12,19 @@ intergrated into the binary of IoT.js.
 ### Usage
 
 To start the debugger-server: `<iotjs binary> --start-debug-server test.js`
+
+It is important to note that optional parameters (such as `--debugger-wait-source` or
+`jerry-debugger-port=...`) should be called after `--start-debug-server` in order to work properly.
```
Comment (LaszloLango) [inline docs/devs/Use-JerryScript-Debugger.md]: `--jerry-debugger-port=...`

**Comment by DanielBallaSZTE** [comment]
@LaszloLango Thanks, I've fixed the mentioned line.

File: docs/devs/Use-JerryScript-Debugger.md [hunk 2]
```diff
@@ -12,6 +12,19 @@ intergrated into the binary of IoT.js.
 ### Usage
 
 To start the debugger-server: `<iotjs binary> --start-debug-server test.js`
+
+It is important to note that optional parameters (such as `--debugger-wait-source` or
+`--jerry-debugger-port=...`) should be called after `--start-debug-server` in order to work properly.
```
Comment (zherczeg) [inline docs/devs/Use-JerryScript-Debugger.md]: called? this is argument list not function call :)
* should be specified after
* should come after

File: docs/devs/Use-JerryScript-Debugger.md [hunk 3]
```diff
@@ -12,6 +12,19 @@ intergrated into the binary of IoT.js.
...
+
+#### Sending source to the debugger remotely
+
+Using the `--debugger-wait-source` you can send a source with the debugger-client itself,
```
Comment (zherczeg) [inline docs/devs/Use-JerryScript-Debugger.md]: The `--debugger-wait-source` makes the client to wait until the source code is sent by the debugger-client,

File: docs/devs/Use-JerryScript-Debugger.md [hunk 4]
```diff
@@ -12,6 +12,19 @@ intergrated into the binary of IoT.js.
...
+being able to specify it on-the-fly. However, only  one source file is supported at the moment.
```
Comment (zherczeg) [inline docs/devs/Use-JerryScript-Debugger.md]: two spaces before `one`?

File: docs/devs/Use-JerryScript-Debugger.md [hunk 5]
```diff
@@ -12,6 +12,19 @@ intergrated into the binary of IoT.js.
...
+In this case, you don't have to pass any file arguments. IoT.js is also capable of resetting the context,
```
Comment (zherczeg) [inline docs/devs/Use-JerryScript-Debugger.md]: The file argument is ignored after this option. (please use passive sentences)

**Comment by DanielBallaSZTE** [comment]
@zherczeg I appreciate the review, and completely agree with your review. Thanks for the suggestions, I've changed the PR according to it.

**Pull request merged**